package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "node-forge",
-  "version": "0.7.7-dev",
+  "version": "0.7.6-1",
```

Comment:
Would rather not have this in the patch since we try to keep the -dev suffix for unreleased code.